src/gui/Layout.h

```diff
@@ -107,45 +113,40 @@ class Layout
 
 private:
 	void checkScroll(GtkAdjustment* adjustment, double& lastScroll);
+
 	void setLayoutSize(int width, int height);
 
 private:
 	XOJ_TYPE_ATTRIB;
 
-	XournalView* view = NULL;	
-
 	LayoutMapper mapper;
 
-	ScrollHandling* scrollHandling = NULL;
+	XournalView* view{};
+	ScrollHandling* scrollHandling{};
+
+	std::vector<int> widthCols;
+	std::vector<int> heightRows;
 
-	double lastScrollHorizontal = -1;
-	double lastScrollVertical = -1;
+	double lastScrollHorizontal{-1};
+	double lastScrollVertical{-1};
```

Comment:
Do not use this initialization routine as it may not be supported by every compiler. 
Stick with `type variable = value;`

Reply:
Mhh strange this is a c++11 feature and all compilers which stick to the standard should support it. It also is (in my opinion) cleaner to read and to write.

Reply:
Well compilers unfortunately are not always compliant. Anyway we adhere to the mentioned standard in the rest of the code so we should not mix these code styles.